lang/en/lang.php

```diff
@@ -131,5 +131,13 @@
         'slug_desc' => 'The URL route parameter used for looking up the topic by its slug. A hard coded slug can also be used.',
         'channel_title' => 'Channel Page',
         'channel_desc' => 'Page name to use for clicking on a channel.'
+    ],
+    'components' => [
+        'general' => [
+            'properties' => [
+                'includeStyles' => 'Enable CSS',
+                'includeStyles_desc' => 'Include css file with default styles for forum'
+            ]
+        ]
```

Comment:
Looking good, can you make sure you add trailing commas as well please?

Reply:
added